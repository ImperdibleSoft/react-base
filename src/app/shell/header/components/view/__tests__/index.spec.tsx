import * as React from 'react';
import { shallow } from 'enzyme';

import View from '..';

describe('Header', () => {
  it('should render a header', () => {
    const component = shallow(
      <View logo="custom-logo.png" logSmartEvent={() => {}} openMenu={''} toggleMenu={() => {}} />
    );

    expect(component).toMatchSnapshot();
  });
});
