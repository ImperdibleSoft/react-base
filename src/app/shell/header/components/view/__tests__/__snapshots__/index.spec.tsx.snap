// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render a header 1`] = `
<Header>
  <HeaderBar>
    <Sidebar
      handleClick={[Function]}
      isOpen={false}
    >
      <ul />
    </Sidebar>
    <LogoLink
      onClick={[Function]}
      to="/"
    >
      <Logo
        src="custom-logo.png"
      />
    </LogoLink>
  </HeaderBar>
  <HeaderSpacer />
</Header>
`;
